### DISCUSSION TITLE
502 Bad Gateway while trying to create a new assignment.

### DISCUSSION THREAD

**Discussion by edmundmiller** [community/community#123183] [GitHub Education]
### Hi everyone,

In my classroom(https://github.com/orgs/community/discussions/113185#discussioncomment-9360064) I'm getting an error whenever trying to create or edit assignments.

Coming from https://github.com/orgs/community/discussions/113185#discussioncomment-9360064

**Comment by slauC113** [comment]
I am getting the same thing when I try to create a new assignment. 

Every time I try to create a new assignment, I keep getting the following issues. This recently started happening as I deleted some assignments and some repositories. However, I can't understand why deleting anything would prevent me from creating a new assignment. So I don't believe this is the issue. 


![image](https://github.com/community/community/assets/164940807/15deb834-151e-402b-b66e-98570c6b1bfa)

**Comment by y0geshx** [comment]
Same error 🥲,
I hope GitHub will resolve this soon 🤞

**Comment by rlalik** [comment]
Similar to mine: https://github.com/orgs/community/discussions/125198
